test/network.bats

```diff
@@ -2,6 +2,14 @@
 
 load helpers
 
+function teardown() {
+	cleanup_ctrs
```

Comment:
I think it's OK, but want to double check.  If you run the cleanup_ctrs or cleanup_pods after a test that didn't create one or the other, it doesn't create an error correct?  I think not, but I'm not 100% sure.

Reply:
@TomSweeneyRedHat correct, it doesn't create an error.  A couple of the tests in network.bats do not create containers, and the tests pass even when cleanup_ctrs() is called.

Reply:
LGTM, thanks for the follow up @dcbw